engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-package-groups.html

```diff
@@ -3,10 +3,10 @@
   <div data-block="header"></div>
   <div data-block="selection-summary"></div>
 
-  <span class="input-group-addon" data-block="search-filter">
+  <div data-block="filters">
     <select ng-model="repository" ng-options="repository.name for (id, repository) in repositories">
```

Comment:
This ends up looking differently styled than say the one on the Errata page. Maybe due to the form-control bootstrap class?